src/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Digest.cs

```diff
@@ -16,27 +16,17 @@ internal static partial class AppleCrypto
         [DllImport(Libraries.AppleCryptoNative, EntryPoint = "AppleCryptoNative_DigestCreate")]
         internal static extern SafeDigestCtxHandle DigestCreate(PAL_HashAlgorithm algorithm, out int cbDigest);
 
-        internal static unsafe int DigestUpdate(SafeDigestCtxHandle ctx, ReadOnlySpan<byte> pbData, int cbData)
-        {
-            fixed (byte* pbDataPtr = &pbData.DangerousGetPinnableReference())
-            {
-                return DigestUpdate(ctx, pbDataPtr, cbData);
-            }
-        }
+        internal static int DigestUpdate(SafeDigestCtxHandle ctx, ReadOnlySpan<byte> pbData, int cbData) =>
+            DigestUpdate(ctx, ref pbData.DangerousGetPinnableReference(), cbData);
```

Comment:
I've been leaving these defined like this under the assumption that eventually we'll just be able to declare the P/Invokes with `ReadOnlySpan<byte>` instead of `ref byte` or `byte*` or `byte[]`.